xds/internal/balancer/cdsbalancer/cdsbalancer.go

```diff
@@ -515,7 +515,7 @@ func (b *cdsBalancer) handleErrorFromUpdate(err error, fromParent bool) {
 // xdsResolver.
 func (b *cdsBalancer) UpdateClientConnState(state balancer.ClientConnState) error {
 	if b.closed.HasFired() {
-		b.logger.Warningf("xds: received ClientConnState {%+v} after cdsBalancer was closed", state)
+		b.logger.Warningf("Received balancer config update after close: %v", pretty.ToJSON(state.BalancerConfig))
```

Comment:
Nit: this is taking away information. Previously this would log the resolver state too. I think saying received ClientConnState after closure is a more correct statement that received balancer config, as our balancer.Balancer API guarantees that no other *operations* (i.e. the full UpdateClientConnState/balancer.ClientConnState data) are triggered after closure, not just specifically the balancer config. If you want the json to be prettier (pun not intended) perhaps something like "received ClientConnState after close: state.ResolverState: %formatting directive for resolver state, state.BalancerConfig: %v, state.ResolverState, pretty.ToJSON(state.BalancerConfig)".

Reply:
`resolver.State` has three fields:
- `Addresses []Address`: In this case, we do not expect to receive any addresses. Hence no value in printing this.
- `ServiceConfig *serviceconfig.ParseResult`: This contains the balancer config (plus possibly some other information). But this type does not print well even with our `pretty.ToJSON()`. Hence no value in printing this.
- `Attributes *attributes.Attributes`: Same here, it does not print well with our `pretty.ToJSON()`. Hence no value in printing this.

My argument about the value of this log statement is very similar to the previous comment. This log statement by itself has low value, but I didn't want to completely get rid of it (in the hope that maybe someday for some event, it might be useful).

Reply:
Discussed offline; info log for whole state (no one feels strongly though), error log with just the received operation after Close() to fail tests since this is an error in the system coding it.